how-to-guides/present-cross-validation-results/scripts/neptune_cross_validation.py

```diff
@@ -0,0 +1,110 @@
+# How to present CV with Neptune
+import neptune.new as neptune
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torchvision import datasets, transforms
+from torch.utils.data import SubsetRandomSampler, DataLoader
+from sklearn.model_selection import KFold
+from statistics import mean
+
+# Step 1: Create a Neptune Run
+run = neptune.init(
```

Comment:
Please, keep order of arguments as they appear in the docs -> standardization.
https://docs.neptune.ai/api-reference/neptune#.init_run